src/commands/svc/restart.ts

```diff
@@ -0,0 +1,49 @@
+import { CliUx, Flags } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+import { SvcProcessResult, ServiceTypes } from './svc-process-result'
+
+export default class Restart extends SvcProcessResult {
+  static strict = false
+  static description = 'Restart running auxiliary services'
+
+  static examples = [
+    '<%= config.bin %> <%= command.id %> --all',
+    '<%= config.bin %> <%= command.id %> ext-service',
+    '<%= config.bin %> <%= command.id %> ext-service1 ext-service2'
+  ]
+
+  static flags = {
+    all: Flags.boolean({
+      description: 'Stops all enabled services in the bundle descriptor'
```

Comment:
Stops -> Restart. Also please check verbs in other flag commands as I see we still have wrong verbs

Reply:
done